src/lib/utils.js

```diff
@@ -431,28 +476,42 @@ function deleteProperties(propertiesToDelete: Array<string>, objectItem: any) {
   return objectItem;
 }
 
-function addGravatarSupport(pkgInfo: any) {
-  if (_.isString(_.get(pkgInfo, 'latest.author.email'))) {
-    pkgInfo.latest.author.avatar = generateGravatarUrl(pkgInfo.latest.author.email);
-  } else {
-    // _.get can't guarantee author property exist
-    _.set(pkgInfo, 'latest.author.avatar', generateGravatarUrl());
+function addGravatarSupport(pkgInfo: Object): Object {
```

Comment:
Isn't there a flow type for `pkgInFo`?

Reply:
I am not able to find. If I use package type flow gives following error:
```
flow] Cannot get `pkgInfoCopy.latest` because property `latest` is missing in object literal [1]. (References: [1])
```